src/manage_sql.c

```diff
@@ -41828,11 +41853,20 @@ create_credential (const char* name, const char* comment, const char* login,
 
   /* Add non-secret data */
   if (login)
-    set_credential_data (new_credential,
-                         "username", login);
-  if (key_public)
```

Comment:
Looks like this was removed.  Is that intentional?

Reply:
No, removing the key_public part was accidental and I've added it again in 2eed1cc.